From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Price <greg@zulip.com>
Date: Tue, 31 May 2022 20:35:47 -0700
Subject: [PATCH] Trivial patch, as placeholder

---
 types/@react-navigation/stack.js.flow | 1 +
 1 file changed, 1 insertion(+)

diff --git a/types/@react-navigation/stack.js.flow b/types/@react-navigation/stack.js.flow
index 253b2f62f..1e60c5cf4 100644
--- a/types/@react-navigation/stack.js.flow
+++ b/types/@react-navigation/stack.js.flow
@@ -2140,3 +2140,4 @@ declare export var GestureHandlerRefContext: React$Context<
   ?GestureHandlerRef,
 >;
 declare export function useGestureHandlerRef(): GestureHandlerRef;
+
-- 
2.30.2

