From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Bobbe <cbobbe@zulip.com>
Date: Fri, 26 Jan 2024 11:43:38 -0800
Subject: [tsflower] Fix a `typeof` to `type` in Int32 import

---
 .../clipboard/dist/NativeClipboardModule.js.flow                | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git types/@react-native-clipboard/clipboard/dist/NativeClipboardModule.js.flow types/@react-native-clipboard/clipboard/dist/NativeClipboardModule.js.flow
index 58fa7b277..ae2537986 100644
--- types/@react-native-clipboard/clipboard/dist/NativeClipboardModule.js.flow
+++ types/@react-native-clipboard/clipboard/dist/NativeClipboardModule.js.flow
@@ -3,7 +3,7 @@
  */
 import type { EmitterSubscription as $tsflower_subst$RN$EmitterSubscription } from 'tsflower/subst/react-native';
 import { type TurboModule } from 'react-native/Libraries/TurboModule/RCTExport';
-import { typeof Int32 } from 'react-native/Libraries/Types/CodegenTypes';
+import { type Int32 } from 'react-native/Libraries/Types/CodegenTypes';
 
 export interface Spec extends TurboModule {
   getString(): Promise<string>;
-- 
2.32.0

